entity-api-spec.yaml

```diff
@@ -1,396 +1,1267 @@
+        last_modified_user_displayname:
+          type: string
+          readOnly: true
+          description: "The name of the person or process which authenticated when the object was last modified."
+        ###### All other shared properties ######
+        entity_type:
+          type: string
+          readOnly: true
+          description: "One of the normalized entity types: Dataset, Collection, Sample, Donor"
+        registered_doi:
+          type: string
+          description: "The doi of a the registered entity. e.g. 10.35079/hbm289.pcbm.487.  This is set during the publication process and currently available for certain Collections and Datasets."
+        doi_url:
+          type: string
+          readOnly: true
+          description: "The url from the doi registry for this entity. e.g. https://doi.org/10.35079/hbm289.pcbm.487"
+        creators:
+          type: array
+          items:
+            $ref: '#/components/schemas/Person'
+          description: "A list of the people who created the entity with full name, email, ORCID iD, institution, etc.. This is analogus to the author list on a publication."
+        contacts:
+          type: array
+          items:
+            $ref: '#/components/schemas/Person'
+          description: "A list of the people who are the main contacts to get information about the entity."
+        description:
+          type: string
+          description: "Free text description of the sample"
+        data_access_level:
+          type: string
+          enum:
+            - consortium
+            - public
+          description: "One of the values: public, consortium."
+        specimen_type:
+          type: string
+          enum:
+            - atacseq
+            - biopsy
+            - blood
+            - cell_lysate
+            - clarity_hydrogel
+            - codex
+            - cryosections_curls_from_fresh_frozen_oct
+            - cryosections_curls_rnalater
+            - ffpe_block
+            - ffpe_slide
+            - fixed_frozen_section_slide
+            - fixed_tissue_piece
+            - flash_frozen_liquid_nitrogen
+            - formalin_fixed_oct_block
+            - fresh_frozen_oct_block
+            - fresh_frozen_section_slide
+            - fresh_frozen_tissue
+            - fresh_frozen_tissue_section
+            - fresh_tissue
+            - frozen_cell_pellet_buffy_coat
+            - gdna
+            - module
+            - nuclei
+            - nuclei_rnalater
+            - organ
+            - organ_piece
+            - other
+            - pbmc
+            - pfa_fixed_frozen_oct_block
+            - plasma
+            - protein
+            - ran_poly_a_enriched
+            - rna_total
+            - rnalater_treated_and_stored
+            - rnaseq
+            - scatacseq
+            - scrnaseq
+            - segment
+            - seqfish
+            - sequence_library
```

Comment:
`sequence_library` appeared twice in the list here. 

 I also noticed that we have two entries of `sequence_library` in the yaml definitions:
- https://github.com/hubmapconsortium/search-api/blob/test-release/src/search-schema/data/definitions/enums/tissue_sample_types.yaml#L79
- https://github.com/hubmapconsortium/search-api/blob/test-release/src/search-schema/data/definitions/enums/tissue_sample_types.yaml#L83

Their descriptions are slightly different with one has lowercased library and the other capitalized.